src/external/modules/view-licences/controller.js

```diff
@@ -100,11 +101,12 @@ async function postLicenceRename (request, h) {
 
   // Rename licence
   const { name } = getValues(form);
-  const { error } = await services.crm.documents.setLicenceName(documentId, name);
+  const rename = !!documentName;
+  const { error } = await services.water.documents.postLicenceRename(documentId, { documentName: name, rename, userName });
```

Comment:
I'm not sure if the { error } = ...  and throwIfError are doing anything here.
For example, if the water service returns a 404 error, I think an error would already be thrown by the services.water.documents.postLicenceRename() call.  So it may be OK to just have:
```
await services.water.documents.postLicenceRename(documentId, { documentName: name, rename, userName });
```
Or if you want to catch it:
```
try {
  await services.water.documents.postLicenceRename(documentId, { documentName: name, rename, userName });
}
catch(err) {
  // do something else
}

Reply:
done